eZ/Bundle/EzPublishCoreBundle/Tests/EventListener/ViewControllerListenerTest.php

```diff
@@ -86,7 +86,7 @@ protected function setUp()
     public function testGetSubscribedEvents()
     {
         $this->assertSame(
-            array( KernelEvents::CONTROLLER => 'getController' ),
+            [KernelEvents::CONTROLLER => ['getController', 10]],
```

Comment:
Again here, why changing it? This is not new code and would eventually lead to backport issues in the future.